src/test/java/seedu/address/testutil/TypicalEntries.java

```diff
@@ -83,6 +84,22 @@
             .withAddress("chicago ave")
             .build();
 
+    // Must have valid link
+    public static final Entry VALID_LINK = new EntryBuilder()
+            .withName("Valid Link")
+            .withPhone("Valid link")
+            .withEmail("file://"
+                    + MainApp.class.getResource("/view/BrowserPanelTest/default.html").toExternalForm().substring(5))
```

Comment:
Will these html files get compiled into the jar properly? We can merge for now but we need to check how this works

Reply:
This particular line only gets the link of the resource, but if you're worried about whether 
html files get compiled into the jar properly, I think they should. I'm following the AB4 example of the default background page, so it should be fine.